Sources/BraintreeShopperInsights/BTCustomerSessionRequest.swift

```diff
@@ -1,7 +1,8 @@
 import Foundation
 
-/// The POST body for the `createCustomerSession` GraphQL API.
-struct BTCustomerSessionRequest: Encodable {
+// swiftlint:disable nesting
+/// A `BTCustomerSessionRequest` that specifies options for the Payment Ready v2 flow.
+struct BTCustomerSessionRequest {
```

Comment:
I _think_ we want this to be public right? This will be the request merchants ultimately pass to us? If so we should docstring the properties. I think we want to be following this format here vs nesting the objects: https://github.com/braintree/braintree_android/pull/1317/files

Reply:
ahh yep that's true, addressed this in a57e5266f83742251a4231505231207a99dae06b